src/alt/store/StoreMixin.js

```diff
@@ -61,6 +61,12 @@ const StoreMixin = {
 
     // You can pass in the constant or the function itself
     const key = symbol[Sym.ACTION_KEY] ? symbol[Sym.ACTION_KEY] : symbol
+    if (this[Sym.LISTENERS].hasOwnProperty(key)) {
+      throw new ReferenceError(
+        `The action ${key.toString()} has already been bound to store` +
```

Comment:
Not that it's that important but should there be a space between store and $(storename)?
ASIDE: just curious why split the string across lines?